udata/app.py

```diff
@@ -206,7 +206,6 @@ def register_extensions(app):
         mail,
         models,
         mongo,
-        notifications,
```

Comment:
I would pay attention when removing this kind of imports this because there are some signals registering in notifications when loading it.

Reply:
Ok, i'll do that 👍